src/archanaNJun20/PalindromeNumber.java

```diff
@@ -0,0 +1,22 @@
+package archanaNJun20;
+
+public class PalindromeNumber {
+	int rev=0;
+	void palindromeNumber(int number){
+		int oNumber=number;
+		while(number>0){
+			int mod=number%10;
+			rev=(rev*10)+mod;
+			number=number/10;
+		}	
+		if(oNumber==rev)
+			System.out.println("Given number : "+ oNumber +" is palindrome");
+		else
+			System.out.println("Given number : "+ oNumber +" is not palindrome");
+	}
+	public static void main(String[] args) {
+		PalindromeNumber palindromeNumber=new PalindromeNumber();
+		palindromeNumber.palindromeNumber(45654);
+		palindromeNumber.palindromeNumber(56789);
```

Comment:
It is not best practice to reuse same variable name with different context.
Always try to use different name for method and variable name to maintain understandability and readability of your program.

Reply:
The code is updated based on the comment. Thank you for your Suggestion.